packages/app-accounts/src/Editor.js

```diff
@@ -135,6 +135,7 @@ class Editor extends React.PureComponent<Props, State> {
               })}
               onChange={this.onChangeName}
               value={editedName}
+              isEditable
```

Comment:
Move it up, alphabetical. (Just makes it a bit easier to spot if you know that things are always sorted)